langchain/agents/helpers.ts

```diff
@@ -32,5 +32,8 @@ export const deserializeHelper = async <
     SerializedLLMChain
   >("llm_chain", data);
   const llmChain = await LLMChain.deserialize(serializedLLMChain);
+
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
```

Comment:
This is throwing a ts-error:

```
Argument of type '({ _type: T; llm_chain?: SerializedLLMChain; llm_chain_path?: string; } & { load_from_llm_and_tools: true; } & U & { llmChain: LLMChain; }) | ({ _type: T; llm_chain?: SerializedLLMChain; llm_chain_path?: string; } & { ...; } & V & { ...; })' is not assignable to parameter of type 'V'.
  '({ _type: T; llm_chain?: SerializedLLMChain; llm_chain_path?: string; } & { load_from_llm_and_tools: true; } & U & { llmChain: LLMChain; }) | ({ _type: T; llm_chain?: SerializedLLMChain; llm_chain_path?: string; } & { ...; } & V & { ...; })' is assignable to the constraint of type 'V', but 'V' could be instantiated with a different subtype of constraint 'AgentInput'.
    Type '{ _type: T; llm_chain?: SerializedLLMChain; llm_chain_path?: string; } & { load_from_llm_and_tools: true; } & U & { llmChain: LLMChain; }' is not assignable to type 'V'.
      '{ _type: T; llm_chain?: SerializedLLMChain; llm_chain_path?: string; } & { load_from_llm_and_tools: true; } & U & { llmChain: LLMChain; }' is assignable to the constraint of type 'V', but 'V' could be instantiated with a different subtype of constraint 'AgentInput'.ts(2345)
```

I am not sure why it was not complaining before.